src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,82 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import React from 'react';
+import { Filter, TodoStates } from '../../types/Filter';
+
+function capitalizer(str: string): string {
+  return str.at(0)?.toUpperCase() + str.slice(1);
+}
```

Comment:
As you use it only for UI, to display in options you can just use CSS property - `text-transform: capitalize;`
https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform